wolfssl/wolfcrypt/mlkem.h

```diff
@@ -343,9 +343,6 @@ WOLFSSL_API int wc_MlKemKey_EncodePrivateKey(MlKemKey* key, unsigned char* out,
 WOLFSSL_API int wc_MlKemKey_EncodePublicKey(MlKemKey* key, unsigned char* out,
     word32 len);
 
-
-#define KyberKey            MlKemKey
```

Comment:
This doesn't look like a redefinition of MlKemKey to me.
KyberKey shouldn't be used anywhere except in these macros below.

Reply:
Prefer the removal of the typed as it is clearer then that the KyberKey type is not meant to be used.